trame_server/state.py

```diff
@@ -238,12 +238,17 @@ def flush(self):
 
                 # Execute state listeners
                 self._state_listeners.add_all(_keys)
+
+                pushed_state = self.translator.reverse_translate_dict(
+                    self._pushed_state
+                )
+
                 for callback in self._state_listeners:
                     if self._hot_reload:
                         if not inspect.iscoroutinefunction(callback):
                             callback = reload(callback)
 
-                    coroutine = callback(**self._pushed_state)
+                    coroutine = callback(**pushed_state)
```

Comment:
you could not figure out the args names?

Reply:
I could, but I wasn't happy with the result.
If we have a callback like this:
```python
def cb(foo, **kwargs):
    pass
```
We get the translated `foo` argument, but all the other `kwargs` are passed untranslated.
I didn't like the inconsistency and decided to keep the initial implementation.

Reply:
I see

Reply:
Then my worry now is in the case you have the following root state

```
{
   "resolution": 5,
   "prefix_resolution": 10,
}
```

What do you get as translated state for a callback on the server with `prefix_`?

`{ resolution: 5 or 10 }`

Reply:
It seems that the reverse translation should only keep the translated keys and the system one.